python/tutorials/01-vector-add.py

```diff
@@ -60,7 +62,8 @@ def add_kernel(x_ptr,  # *Pointer* to first input vector.
 def add(x: torch.Tensor, y: torch.Tensor):
     # We need to preallocate the output.
     output = torch.empty_like(x)
-    assert x.is_cuda and y.is_cuda and output.is_cuda
+    is_dvc = f"is_{DEVICE}"
+    assert getattr(x, is_dvc) and getattr(y, is_dvc) and getattr(output, is_dvc)
```

Comment:
Do `x.device == DEVICE` here and in the other places where you use `getattr`.

Reply:
Could it be so: `x.device.type == DEVICE`? `x.device == DEVICE` doesn't work.

Reply:
yep, that's better, yes

Reply:
done